src/chains-config/metadata-consts/index.ts

```diff
@@ -0,0 +1,74 @@
+import {
+	BlockWeightStore,
+	ExtBaseWeightValue,
+	MetadataConsts,
+	PerClassValue,
+} from '../../types/chains-config';
+import { kusamaDefinitions } from './kusamaConsts';
+import { polkadotDefinitions } from './polkadotConsts';
+
+/**
+ * Creates an object that orders each runtime to their appropriate weight data.
+ *
+ * Each runtime imports their own data which is called Definitions here. Definitions
+ * are arrays that store objects which group a set of runtimes version connected
+ * to their classification of extrinsic weight data.
+ *
+ * Example return object:
+ * {
+ * 	   ...
+ *     '24': { extrinsicBaseWeight: 125000000 },
+ *     '25': { extrinsicBaseWeight: 125000000 },
+ *     '26': { extrinsicBaseWeight: 125000000 },
+ *     '27': { blockWeights: { perClass: [Object] } },
+ *     '28': { blockWeights: { perClass: [Object] } }
```

Comment:
```suggestion
 *     24: { extrinsicBaseWeight: 125000000 },
 *     25: { extrinsicBaseWeight: 125000000 },
 *     26: { extrinsicBaseWeight: 125000000 },
 *     27: { blockWeights: { perClass: [Object] } },
 *     28: { blockWeights: { perClass: [Object] } }
```
These are numbers now :)